framework/common/utils.cpp

```diff
@@ -109,26 +103,26 @@ void screenshot(RenderContext &render_context, const std::string &filename)
 	swizzle          = std::find(bgr_formats.begin(), bgr_formats.end(), src_image_view.get_format()) != bgr_formats.end();
```

Comment:
it makes sense to just define swizzle here as line 99 definition with setting to false is never read and proves unused until here.

Reply:
Done